docs/development/development.md

```diff
@@ -70,6 +70,12 @@ make build GOOS=linux GOARCH=amd64
 make test
 ```
 
+### Generate CLI docs
+
+```bash

```

Comment:
Can we add the steps here on what is needed to checkin the docs when new command is added or existing one is changed?